tests/myst/exec-package-ext2/Makefile

```diff
@@ -29,8 +30,8 @@ $(TMPDIR):
 export MYST_ROOTFS_PATH=$(ROOTFS)
 
 tests:
-	$(RUNTEST) $(MYST) exec-sgx --pubkey=$(PUBKEY) $(ROOTFS) /bin/hello
-	$(RUNTEST) ./myst/bin/hello
+	$(RUNTEST) $(MYST) exec-sgx $(IDMAPPING) --pubkey=$(PUBKEY) $(ROOTFS) /bin/hello
```

Comment:
largely dependent on what is being done. This test is using ext2. Identity in enclave is root, identity of host was not... when we go to open the file we will try to set the uid to root in the host before running. This will fail unless running as root, or unless we do the mapping.

Reply:
Does this PR solve the problem w/o id mapping? https://github.com/deislabs/mystikos/pull/268

Reply:
not at all. the implemented file operations will try to impersonate the enc identity and will fail without mapping.
sockets will have the same problem going forwards when id is mapped to them too.